frontend/src/components/ThemedInput/ThemedNumberInput.tsx

```diff
@@ -43,7 +43,7 @@ function ThemedNumberInput({
 			className={inputClass}
 			type="number"
 			value={content}
-			disabled={disabled}
+			readOnly={disabled}
```

Comment:
interesting! How does readOnly compare from aria-disabled? I think we're using aria-disabled elsewhere to indicate that something is disabled, but still tabbable.

Reply:
if you have aria-disabled you need to add javascript so that you can't use the input. 
readonly already does that and is announced by screen readers. 
the themedTextInput already uses readOnly. We're using aria-disabled on the buttons.